lib/src/mt_main.h

```diff
@@ -985,7 +1005,14 @@ bool mt_is_valid_socket(struct mtl_main_impl* impl, int soc_id);
 static inline uint8_t mt_start_queue(struct mtl_main_impl* impl, enum mtl_port port) {
   return impl->user_para.xdp_info[port].start_queue;
 }
-
+#ifndef WINDOWSENV
+static inline bool mt_has_phc2sys_service(struct mtl_main_impl* impl) {
```

Comment:
Can we just use `WINDOWSENV` for the build failed function? We should avoid to build control macro usage.

```
#ifdef WINDOWSENV
ptp_adj_system_clock_time() {
  err("%s(%d), not supported for windows\n", __func__, ptp->port);
}
ptp_adj_system_clock_freq {
  err("%s(%d), not supported for windows\n", __func__, ptp->port);
}
#else
ptp_adj_system_clock_time() {
  ...
}
ptp_adj_system_clock_freq {
  ...
}
#endif
```

Reply:
> Can we just use `WINDOWSENV` for the build failed function? We should avoid to use too much build control macro.
> 
> ```
> #ifdef WINDOWSENV
> ptp_adj_system_clock_time() {
>   err("%s(%d), not supported for windows\n", __func__, ptp->port);
> }
> ptp_adj_system_clock_freq {
>   err("%s(%d), not supported for windows\n", __func__, ptp->port);
> }
> #else
> ptp_adj_system_clock_time() {
>   ...
> }
> ptp_adj_system_clock_freq {
>   ...
> }
> #endif
> ```

DONE.